DuckDuckGo/AutofillPixelReporter.swift

```diff
@@ -101,14 +115,29 @@ final class AutofillPixelReporter {
         for pixel in pixels {
             switch pixel {
             case .autofillLoginsStacked:
-                let bucket = (try? secureVault?.accountsCountBucket()) ?? ""
-                DailyPixel.fire(pixel: pixel, withAdditionalParameters: [PixelParameters.countBucket: bucket])
+                if let count = try? vault()?.accountsCount() {
+                    DailyPixel.fire(pixel: pixel, withAdditionalParameters: [PixelParameters.countBucket: accountsBucketNameFrom(count: count)])
+                }
             default:
                 DailyPixel.fire(pixel: pixel)
             }
         }
     }
 
+    public func accountsBucketNameFrom(count: Int) -> String {
```

Comment:
After discussion today we changed the bucketing rules but don't want to impact the temporary passwords survey in progress so made this change locally here for now and will update BSK later when removing the survey